src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/BrotliStream.cs

```diff
@@ -72,6 +82,13 @@ protected override void Dispose(bool disposing)
             }
         }
 
+        /// <summary>Asynchronously releases the unmanaged resources used by the <see cref="System.IO.Compression.BrotliStream" />.</summary>
+        /// <returns>A task that represents the asynchronous dispose operation.</returns>
+        /// <remarks><![CDATA[
+        /// The `DisposeAsync` method lets you perform a resource-intensive dispose operation without blocking the main thread. This performance consideration is particularly important in a [!INCLUDE[win8_appname_long](~/includes/win8-appname-long-md.md)] app or [!INCLUDE[desktop_appname](~/includes/desktop-appname-md.md)] app where a time-consuming stream operation can block the UI thread and make your app appear as if it is not working. The async methods are used in conjunction with the `async` and `await` keywords in Visual Basic and C#.
```

Comment:
Would devs know how to edit these INCLUDE markers?

Reply:
No. We will have to publish new guidance for when devs want to add links to external repos.

Since our aim in the near future is to add comments in triple slash, then the Docs team will help review comments here, and they'll be able to provide guidance whenever necessary.

Reply:
It might be good to clean up the dotnet-api-docs repo first, e.g. replacing all INCLUDEs and deleting multiple successive newlines. Also trailing and leading whitespace.